### PR TITLE
SHARE-527: Media Library Search Fix

### PR DESCRIPTION
**Pull request by schaubes**
---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [x] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
Update media library to use new API and implement search

### Tests - additional information
No additional information

### CODE REVIEW

File: assets/js/custom/MediaLib.js [hunk 1]
```diff
@@ -35,19 +35,39 @@ export function MediaLib (packageName, mediaSearchPath, flavor, assetsDir,
       showTopBarDefault()
     }
 
+    // accepted flavors
+    const acceptedFlavors = [
+      'pocketcode'
+    ]
+    if ($.inArray(flavor, acceptedFlavors) === -1) {
```
Comment (dmetzner) [inline assets/js/custom/MediaLib.js]: It would be great if we could get rid of jquery and its overhead.
you could use indexOf here

**Review by dmetzner — APPROVED** [review]

**Review by dmetzner — CHANGES_REQUESTED** [review]
https://pr3072-web.catrobat.ist.tugraz.at/app/media-library/looks/search/P

Looks like the counter does not work as expected?
image

File: assets/js/custom/MediaLib.js (hunk 1 above)
Comment (dmetzner) [reply assets/js/custom/MediaLib.js]: Note: no need to change it now; we can remove jquery in a separate user story

**Comment by schaubes** [comment]
fixed search count and the jquery snippet, plus some css to enable the media queries again

**Review by dmetzner — APPROVED** [review]

**Pull request merged**